client/coral-admin/src/components/BanUserDialog.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import {Dialog} from 'coral-ui';
+import Button from 'coral-ui/components/Button';
+
+import styles from './BanUserDialog.css';
+
+import I18n from 'coral-framework/modules/i18n/i18n';
+import translations from '../translations';
+const lang = new I18n(translations);
+
+const BanUserDialog = ({open, handleClose, onClickBanUser, user}) => {
```

Comment:
so I think the lines below are trying to set defaults. You can do this cleanly by changing the line above to 

`const BanUserDialog = ({open, handleClose, onClickBanUser, user = {}}) => {`

this sets `user` as always defined so...